GPU/GPUTracking/TPCClusterFinder/clusterFinderDefs.h

```diff
@@ -94,7 +94,8 @@ namespace GPUCA_NAMESPACE
 namespace gpu
 {
 
-using Timestamp = short;
+using TPCTime = int32_t;
```

Comment:
Could you move these definitions into a namespace or class, TPCCF or so, such that they are not globally visible in o2::gpu?

Reply:
Moved them into namespace `o2::gpu::tpccf`.